src/content/contributing/translation-program/index.md

```diff
@@ -22,6 +22,12 @@ If you want to get involved and help us grow the global Ethereum community by tr
   <a href="https://ethereum.org/en/contributing/#poap">More on POAPs</a>
 </InfoBanner>
 
+<br/>
+
+<InfoBanner shouldCenter emoji=":tada:">
+  <a href="https://ethereum.org/en/contributing/translation-program/acknowledgements/">Translator Acknowledgements!</a>
```

Comment:
I'm noticing we use this incorrectly elsewhere as well but this should use the Link component so we can use relative links here vs. hardcoding the production link. i.e.

`<Link to="contributing/translation-program/acknowledgements/">Translator Acknowledgements!</Link>`